kubespawner/spawner.py

```diff
@@ -138,6 +138,7 @@ def __init__(self, *args, **kwargs):
         # runs during both test and normal execution
         self.pod_name = self._expand_user_properties(self.pod_name_template)
         self.pvc_name = self._expand_user_properties(self.pvc_name_template)
+        self.working_dir = self._expand_user_properties(self.working_dir)
```

Comment:
Tests are failing. Looks like because the default working_dir is None, so need to check `if self.working_dir` here before rendering the template.